gravitee-apim-e2e/lib/fixtures/management/ApisFaker.ts

```diff
@@ -22,13 +22,13 @@ import { NewApiEntity } from '@management-models/NewApiEntity';
 import faker from '@faker-js/faker';
 import { NewRatingEntity } from '@management-models/NewRatingEntity';
 import { RatingInput } from '@portal-models/RatingInput';
-import { Plan } from '@management-models/Plan';
 import { PrimaryOwnerEntity } from '@management-models/PrimaryOwnerEntity';
+import { PlanEntity } from '@management-models/PlanEntity';
 
 export interface ApiImportEntity extends ApiEntity {
   members?: Array<any>;
   pages?: Array<PageEntity>;
-  plans?: Array<Plan>;
+  plans?: Array<PlanEntity>;
```

Comment:
This one was strange no?

Reply:
If I remember, it's a rename to avoid conflicts in generated SDK.
@a-cordier or @marcambier are you aware of this ?